src/pages/Index.vue

```diff
@@ -531,10 +537,17 @@
 	}
 
 	.nav {
-		display    : flex;
```

Comment:
You can hide `.nav` when the screen is too small; we have the burger menu to access the navigation links